jwt.go

```diff
@@ -37,7 +217,18 @@ func New(method SigningMethod) *Token {
 			"typ": "JWT",
 			"alg": method.Alg(),
 		},
-		Claims: make(map[string]interface{}),
+		Claims: Claims{},
```

Comment:
Changed the default to use the structured object rather than the map.

This makes the standard behaviour a little more structured.